__tests__/app.test.js

```diff
@@ -237,7 +237,32 @@ describe("POST /api/articles/:article_id/comments", () => {
       .expect(404)
       .then((response) => {
         const msg = response.body.msg;
-        expect(msg).toBe("entry does not exist");
+        expect(msg).toBe("3432432 does not exist in articles");
+      });
+  });
+});
+describe("PATCH /api/articls/:article_id", () => {
+  test("PATCH:200 responds with appropriate status and sends the updated article to the client", () => {
+    const input = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(input)
+      .expect(200)
+      .then((response) => {
+        const article = response.body.article;
+        expect(article).toEqual(
+          {
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 101,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          },
+        );
```

Comment:
So happy path is working. Where are the tests for patching to an article that doesn't exist, patch to an invalid article id, patching with incorrect keys, patching with incorrect data type. There is a lot missing on this push.

Reply:
Hi Verity, yeah I'm aware. I only pushed as I'd made changes on this branch for hosting and I had some merge conflicts earlier today to I just thought I'd push what was on this branch and create a new one for the rest of task 8